keyboards/scorpius_m32/config.h

```diff
@@ -0,0 +1,32 @@
+
+// Copyright 2022 Vencer-Co (@Vencer-Co)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x195D
```

Comment:
This vender ID is in use, and needs to be changed. 
https://www.the-sz.com/products/usbid/index.php?v=0x195D&p=&n=

Generally, anything above 0x4000 is good, but the higher, the better.

Reply:
The VENDOR_ID 0x195D is belongs to my company.